### ISSUE TITLE
List аccessibility improvements

### ISSUE THREAD

**Issue by desig9stein** [IgniteUI/igniteui-webcomponents#472]
## Description  
The screen reader tells you the number of list items but not which item you are on, also it does not read the text inside the item.
